test/unit/resources/etc_fstab_test.rb

```diff
@@ -0,0 +1,96 @@
+# encoding: utf-8
+# author: Matthew Dromazos
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::Fstab' do
+  let(:resource) { load_resource('etc_fstab') }
+  it 'Verify etc_hosts filtering by `device_name`'  do
+    entries = resource.where { device_name == '/dev/mapper/vg1-lv_root' }
+    _(entries.mount_point).must_equal ['/']
+    _(entries.file_system_type).must_equal ['xfs']
+    _(entries.mount_options).must_equal [['defaults', 'x-systemd.device-timeout=0']]
+    _(entries.dump_options).must_equal [0]
+    _(entries.file_system_options).must_equal [0]
+  end
+
+  it 'Verify etc_hosts filtering by `mount_point`'  do
```

Comment:
I actually don't think you need this test, of the tests starting at lines 27, 36, 45, or 57. These tests are all fundamentally testing FilterTable, not really testing your custom resource.

These are the tests I would want to see:
 * basic support of parsing the file
 * parsing an entry where mount_options is a single item
 * parsing an entry where mount_options has multiple items.
 * test that home_mount_options returns something when there is a /home mount configured
 * test that home_mount_options returns nil when there isn't a /home mount configured
 * that that nfs_file_systems returns the correct entries